docs/learn-governance.md

```diff
@@ -230,8 +231,9 @@ The conviction multiplier increases the vote multiplier by one every time the lo
 |      16      |        5        |
 |      32      |        6        |
 
-The maximum number of lock periods is set to 6 and the lock period is 30 days on Polkadot and eight
-days on Kusama.
+The maximum number of "doublings" of the lock period is set to 6 (and thus 32 lock periods in
+total), and the lock period is 28 days on Polkadot and 8 days on Kusama. Only doublings are allowed;
```

Comment:
Do I understand correctly that one period is always 28 days? That means when I lock for 4 Periods, I have my tokens locked for 112 days? If so, I'd propose to change
"and the lock period is 28 days" to "and one lock period is 28 days..."

Reply:
And I still don't fully get the idea of that doubling. If I want to vote for something is it not that I say:

 "I have 100 DOTs and I want to vote for proposal XY and lock it for 8 periods." 

Then I sign the transaction and basically vote with 400 tokens, but they are locked for 8 * 224 days?

Or is it that I lock my tokens, vote and with increasing time I don't unlock them, they gain more weight? But then at what point are the votes counted? And what happens for things to vote on, which are in the immediate future. I just can't utilize on certain multipliers?

Reply:
1. Right, one period is always 28 days. I'll update to say "one"

2. Your vote is "worth" 400 DOT and they count immediately.  Your 100 DOT will be locked for 8 * 28 days after the vote is over.  You can't unlock these beforehand; the democracy lock will exist until after 8 * 28 days is over.

3. Yes, you can still vote with your "heavy" tokens.

Reply:
I'll write some stuff in tomorrow to clarify these.

Reply:
ah okay great, then it's actually easier than I thought and could be clarified :) thanks